.github/workflows/volume_group_operator_ci.yaml

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
if we want to contribute this repo, do we want this PR in this repo?

Reply:
sure why not? and this is something we can change in the future, because it's our ci, not a functional thing

Reply:
because, as you said, it's our ci.. but if it's easy to move it later if we need you can resolve this

Reply:
it's easy to remove from it any CSI references in the future and then it will be more general